README.md

```diff
@@ -1,5 +1,149 @@
 # Peggo
 
+<!-- markdownlint-disable MD041 -->
+
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://img.shields.io/badge/repo%20status-WIP-yellow.svg?style=flat-square)](https://www.repostatus.org/#wip)
+[![GoDoc](https://img.shields.io/badge/godoc-reference-blue?style=flat-square&logo=go)](https://godoc.org/github.com/umee-network/peggo)
+[![Go Report Card](https://goreportcard.com/badge/github.com/umee-network/peggo?style=flat-square)](https://goreportcard.com/report/github.com/umee-network/peggo)
+[![Version](https://img.shields.io/github/tag/umee-network/peggo.svg?style=flat-square)](https://github.com/umee-network/peggo/releases/latest)
+[![License: Apache-2.0](https://img.shields.io/github/license/umee-network/peggo.svg?style=flat-square)](https://github.com/umee-network/peggo/blob/main/LICENSE)
+[![Lines Of Code](https://img.shields.io/tokei/lines/github/umee-network/peggo?style=flat-square)](https://github.com/umee-network/peggo)
+[![GitHub Super-Linter](https://img.shields.io/github/workflow/status/umee-network/peggo/Lint?style=flat-square&label=Lint)](https://github.com/marketplace/actions/super-linter)
+
 Peggo is a Go implementation of the Peggy (Gravity Bridge) Orchestrator originally
 implemented by [Injective Labs](https://github.com/InjectiveLabs/). Peggo itself
 is a fork of the original Gravity Bridge Orchestrator implemented by [Althea](https://github.com/althea-net).
+
+
+## Table of Contents
+
+- [Dependencies](#dependencies)
+- [Installation](#installation)
+- [How to run](#how-to-run)
+- [How it works](#how-it-works)
+
+## Dependencies
+
+- [Go 1.17+](https://golang.org/dl/)
+
+## Installation
+
+To install the `peggo` binary:
+
+```shell
+$ make install
+```
+
+## How to run
+### Setup
+
+First we must register the validator's Ethereum key. This key will be used to
+sign claims going from Ethereum to Umee and to sign any transactions sent to
+Ethereum (batches or validator set updates).
+
+```shell
+$ peggo tx register-eth-key \
+  --cosmos-chain-id="..." \
+  --cosmos-grpc="tcp://..." \
+  --tendermint-rpc="http://..." \
+  --cosmos-keyring=... \
+  --cosmos-keyring-dir=... \
+  --cosmos-from=... \
+  --eth-pk=$ETH_PK
+```
+
+### Run the orchestrator
+
+```shell
+$ peggo orchestrator \
+  --eth-pk=$ETH_PK \
+  --eth-rpc=$ETH_RPC \
+  --relay-batches=true \
+  --relay-valsets=true \
+  --eth-chain-id=... \
+  --cosmos-chain-id=... \
+  --cosmos-grpc="tcp://..." \
+  --tendermint-rpc="http://..." \
+  --cosmos-keyring=... \
+  --cosmos-keyring-dir=... \
+  --cosmos-from=...
+```
+
+### Send a transfer from Umee to Ethereum
+
+This is done using the command `umeed tx peggy send-to-eth`, use the `--help`
+flag for more information.
+
+If the coin doesn't have a corresponding ERC20 equivalent on the Ethereum
+network, the transaction will fail. This is only required for Umee originated
```

Comment:
Any cosmos native asset, including IBC tokens ;-)

```suggestion
network, the transaction will fail. This is only required for Cosmos originated
```

Reply:
Also, should we mention the helpful command `peggo bridge deploy-erc20`?

Reply:
Got it 👌